test/json/scatterDataWithFourNames.json

```diff
@@ -0,0 +1,44 @@
+{
+    "data": [
+        {
+            "name": "Nabisco",
```

Comment:
I have a list of names to add in the demos, all of them pretty brite:

Reply:
radiating
reflecting
shining
sunshine
vivid
brilliant
blazing
brilliant
dazzling
flashing
glistening
glittering
intense
luminous
radiant
shimmering
shiny
silvery
sparkling
sunny
vivid